README.md

```diff
@@ -73,12 +45,15 @@ If authentication is enabled, this service needs to be registered with [Azure Ap
 
 The following environment variables need to be set:
 
-- AZUREID_CLIENT_ID
-- AZUREID_TENANT_ID
-- AZUREID_CLIENT_SECRET
+| Environment Variable  | Description |
```

Comment:
Would change Environment Variable to Name to be consistent w/ table above -- also state in text above tht these are env vars